packages/commonwealth/client/scripts/views/pages/new_proposal/new_proposal_form.ts

```diff
@@ -706,9 +712,9 @@ const NewProposalForm = {
           hasTopics &&
             m(TopicSelector, {
               topics: app.topics.getByCommunity(app.chain.id),
-              updateFormData: (topicName: string, topicId?: number) => {
-                vnode.state.form.topicName = topicName;
-                vnode.state.form.topicId = topicId;
+              updateFormData: (topic: Topic) => {
```

Comment:
A little worried about this call -- do we know it will be a `Topic`?

Reply:
Here I was just following the types. `Topic` is coming from `TopicSelector`, not from the form.